lib/ts/fetch.ts

```diff
@@ -107,7 +107,7 @@ export class FrontToken {
         logDebugMessage("FrontToken.getTokenInfo: called");
```

Comment:
also need a function (exposed via the index file) to get the access token in case of header based auth. This would be especially useful for users who are not using a library which has our interceptors, or are using stuff like websockets or trpc